h2o-algos/src/main/java/hex/glm/GLMModel.java

```diff
@@ -40,6 +41,16 @@ public GLMModel(Key selfKey, GLMParameters parms, GLM job, double [] ymu, double
     _nullDOF = nobs - (parms._intercept?1:0);
   }
 
+  void initDefaultParam() {
+    if (_parms._fold_assignment == Model.Parameters.FoldAssignmentScheme.AUTO) {
```

Comment:
this code is repeated over and over, maybe moved it to Model.class or some helper class:

Reply:
moved to EffectiveParametersUtils